src/sources/vector/mod.rs

```diff
@@ -150,8 +183,14 @@ impl SourceConfig for VectorConfig {
         Ok(Box::pin(source))
     }
 
-    fn outputs(&self, _: LogNamespace) -> Vec<Output> {
-        vec![Output::default(DataType::all())]
+    fn outputs(&self, global_log_namespace: LogNamespace) -> Vec<Output> {
+        let log_namespace = global_log_namespace.merge(self.log_namespace);
+
+        let schema_definition = BytesDeserializerConfig
```

Comment:
Should be using the [`Native` decoder](https://github.com/vectordotdev/vector/blob/master/lib/codecs/src/decoding/format/native.rs), no?